testutil/promrated/rated.go

```diff
@@ -90,6 +90,8 @@ func getValidationStatistics(ctx context.Context, ratedEndpoint string, validato
 			backoff()
 
 			continue
+		} else if res.StatusCode == http.StatusUnauthorized {
+			panic("unauthorized")
```

Comment:
suggest handle this error rather than panic

Reply:
How would you handle it? The intention behind the panic is to start a crash loop so we can know that it isn't working. Nothing will change if we repeatedly try do this.

Reply:
crashloop is not a way to do monitoring, prefer logs + metrics and then grafana alerts

Reply:
OK I added a metric for counting rated api errors if that works better 👍🏼